build.js

```diff
@@ -14,13 +14,20 @@ import { esbuildStylesPlugins } from './pkg/lib/esbuild-common.js';
 import { eslintPlugin } from './pkg/lib/esbuild-eslint-plugin.js';
 import { stylelintPlugin } from './pkg/lib/esbuild-stylelint-plugin.js';
 
+const production = process.env.NODE_ENV === 'production';
 const useWasm = os.arch() !== 'x64';
 const esbuild = (await import(useWasm ? 'esbuild-wasm' : 'esbuild')).default;
 
-const production = process.env.NODE_ENV === 'production';
-const watchMode = process.env.ESBUILD_WATCH === "true";
-// linters dominate the build time, so disable them for production builds by default, but enable in watch mode
-const lint = process.env.LINT ? (process.env.LINT !== '0') : (watchMode || !production);
+const parser = (await import('argparse')).default.ArgumentParser();
+parser.add_argument('-r', '--rsync', { help: "rsync bundles to ssh target after build", metavar: "HOST" });
+parser.add_argument('-w', '--watch', { action: 'store_true', help: "Enable watch mode", default: process.env.ESBUILD_WATCH === "true" });
+parser.add_argument('-e', '--no-eslint', { action: 'store_true', help: "Disable eslint linting", default: process.env.LINT ? process.env.LINT === '0' : production });
+parser.add_argument('-s', '--no-stylelint', { action: 'store_true', help: "Disable stylelint linting", default: process.env.LINT ? process.env.LINT === '0' : production });
```

Comment:
Very long lines, and DRY. Please keep it outside as a const (like in old line 23) and rename it to `lint_default`.